### ISSUE TITLE
Classify programs and competitions.

### ISSUE THREAD

**Issue by TouchstoneTheDev** [avinash201199/Competitions-and-Programs-List#517]
Some programs  expires, i guess competitions mostly.
I really appreciate the project.
it needs to updated

**Comment by avinash201199** [comment]
> Some programs expires, i guess competitions mostly. I really appreciate the project. it needs to updated and it would be good to have webpage though

yup the website needs to be updated , if you can work on something then please let me know
